app/services/datetime_parser.rb

```diff
@@ -11,13 +11,13 @@ def parse
       check_date_is_valid
       check_time_is_valid
 
-      Time.new(
+      Time.find_zone("London").local(
```

Comment:
Can you squash this up to commit 3d3905f60cd40b1b9511009b23aa65651267f439

Reply:
Whoops 👍